[WIP] housing-assistance

rroueche opened the pull request:
From this issue: https://github.com/department-of-veterans-affairs/vets.gov-content/issues/236

Commit:
stubbing out housing assistance pages

rroueche commented:
@bethpotts ready for you to pour the content in!

bethpotts commented:
@rroueche Thank you very kindly for stubbing these out! @DanielleSoCompany Here's the PR and branch (content/wip/housing-assistance) for uploading content.

bethpotts commented:
@rroueche Wanted to confirm where we landed with the url for the apply page: apply-for-certificate-of-eligibility. Sound right? If so, I think the folder name needs to change.

rroueche commented:
@bethpotts yep you are correct! apply-for-certificate-of-eligibility i'll do that now and will confirm here when it's ready

Commit:
updating Apply file name

k8reimann committed:
housing assistance index pour

index housing assistance copy - please review!

k8reimann committed:
poured content for home loans

Note use of PK coding - these should be cards but coding was not included in the page layout. Please advise.

k8reimann committed:
poured content for va-backed loans

DanielleSoCompany committed:
changing "adaptive housing" from PK to Card

k8reimann committed:
poured content for purchase loan

outstanding: callout box and callout button info from copy deck

k8reimann committed:
poured content for NADL

outstanding: button from copy deck

k8reimann committed:
poured content for IRRRL

can we check line 67? no ** in copy deck but I noticed all other steps have bolded content to lead

k8reimann committed:
poured content for cash-out refinance loan

outstanding: button (apply for CEO)

rroueche committed:
fixing bullet md

DanielleSoCompany committed:
fixing piano keys

DanielleSoCompany committed:
fixing piano keys

DanielleSoCompany committed:
fixing link

changed /housing-assistance/home-loans/apply-for-coe to /housing-assistance/home-loans/apply-for-certificate-of-eligibility

DanielleSoCompany committed:
Adding back slashes to links

DanielleSoCompany committed:
Pouring in content

DanielleSoCompany committed:
Pouring in content

DanielleSoCompany committed:
updating template

DanielleSoCompany committed:
Pouring in content

DanielleSoCompany committed:
working on table

DanielleSoCompany committed:
updating service-connected disability definition

DanielleSoCompany committed:
updating template

melwoodard committed:
Add list formatting to table

DanielleSoCompany committed:
pouring in content

DanielleSoCompany committed:
ready for beth

- Added missing period in first piano key
- Added template
Note: Alignment of page is off

DanielleSoCompany committed:
ready for Beth

- fixed description of third piano key

Beth: Can you give me your thoughts on the final link on this page?
[Learn more about VA-backed home loans](http://www.benefits.va.gov/home-loans/).

The link is broke and I'm wondering why we wanted to link here and if there's somewhere else we should link to?

DanielleSoCompany committed:
updating template

DanielleSoCompany committed:
updating template

DanielleSoCompany committed:
updating template

DanielleSoCompany committed:
updating template

DanielleSoCompany committed:
adding tables

Commit:
Update eligibility table formatting

DanielleSoCompany committed:
fixing template

DanielleSoCompany committed:
fixing template

DanielleSoCompany committed:
fixing template

DanielleSoCompany committed:
fixing template

DanielleSoCompany committed:
fixing template

DanielleSoCompany committed:
fixing template

DanielleSoCompany committed:
fixing template

DanielleSoCompany committed:
fixing template

DanielleSoCompany committed:
removing blue box

DanielleSoCompany committed:
adding button and fixing title

Beth:  few things here:

1) The www.benefits.va.gov/home-loans/ link appears to be broken. Let's call this out to SMEs and ask about a redirect. 
2) Made the video links calls to action and added to "If you need more help:" links because that seemed like a good fit (rather than a right-column callout). Let me know if you disagree.
3) We have in the deck that there should be an "Apply for your COE" button callout. I'll ask Rachael about adding this. Thanks!

DanielleSoCompany committed:
fixing piano key title

DanielleSoCompany committed:
fixing piano key title

DanielleSoCompany committed:
fixing subway map

DanielleSoCompany committed:
update

DanielleSoCompany committed:
adding <br> in blue box

DanielleSoCompany committed:
fixing subway map

DanielleSoCompany committed:
updating card title to match apply for coe page

DanielleSoCompany committed:
adding <br> and fixing link

bethpotts committed:
Proof: housing-assistance-landing page

bethpotts committed:
Proof: adaptive-housing-grants

melwoodard commented on content/pages/housing-assistance/adaptive-housing-grants.md:
```diff
@@ -0,0 +1,84 @@
+
+You may be able to get an SAH grant if you’re using the grant money to buy, build, or change your permanent home (a home you plan to live in for a long time) if:
+
+-	You own or will own the home, **and** 
```
@bethpotts this is one of those formatting errors Rachael mentioned from copy/pasting from Word - even though when you edit the file the bullet and spacing looks correct, you can see here in the code output that there's a bunch of extra space between the bullet and the text. That unfortunately causes the sub-bullets to not indent correctly.

I'll fix this here, but hopefully this helps with troubleshooting - when things are looking wonky, try deleting and retyping the bullets or numbers in a list.

melwoodard committed:
Update bullet formatting

bethpotts committed:
Proof: home-loans

bethpotts committed:
Proof: va-backed-loans

bethpotts committed:
Proof: purchase-loan

bethpotts committed:
Proof 2: purchase-loan

bethpotts committed:
Proof: cash-out-refinance

rroueche committed:
Housing callouts (#5409)

* adding logic for housing callout

* updating template for callouts

* updating from md to html

bethpotts committed:
Proof 2: cash-out-refinance

rroueche committed:
updating callout

bethpotts committed:
Proof: nadl

bethpotts committed:
Proof: irrrl

bethpotts committed:
Proof: trouble-making-payments

bethpotts committed:
Proof 2: trouble-making-payments

Commit:
Move service requirements into accordions

bethpotts committed:
Proof: eligibility

bethpotts committed:
Proof 2: eligibility

bethpotts committed:
Proof: apply-for-certificate-of-eligibility

bethpotts committed:
fixed capping in piano keys: va-backed-loans

melwoodard committed:
Change COE table to plain text

melwoodard committed:
Add green apply button to COE page

melwoodard committed:
Add headings + green apply button to cash out page

melwoodard committed:
Add green apply button to eligibility page

melwoodard committed:
Headings and green apply button on purchase loan

melwoodard committed:
Add green apply button to NADL

Commit:
revert button links to COE to text links

melwoodard committed:
Merge pull request #5574 from department-of-veterans-affairs/housing-design-updates

Housing content design updates (proposed)

Commit:
Merging master

Commit:
Merge branch 'master' into housing-template-updates

Commit:
housing template updates

melwoodard committed:
Change text links in Apply for COE to relatedlinks

peggygannon committed:
Updated text on related links

peggygannon committed:
Added text to relatedlinks

peggygannon committed:
Restored text for related links

bethpotts committed:
SME feedback: va-backed-loans

bethpotts committed:
SME feedback: apply-for-certificate-of-eligibility

bethpotts committed:
SME feedback: trouble-making-payments

bethpotts committed:
SME feedback: trouble-making-payments

bethpotts committed:
SME feedback: trouble-making-payments

bethpotts committed:
SME feedback: trouble-making-payments

bethpotts committed:
SME feedback: trouble-making-payments

melwoodard committed:
Merge branch 'content/wip/housing-assistance' into housing-template-updates

melwoodard committed:
Update process list classes on apply for COE page

melwoodard committed:
Revert to remove duplicate links on apply COE page

bethpotts committed:
SME feedback: va-backed-loans

bethpotts committed:
SME feedback: purchase-loan

bethpotts committed:
SME feedback: purchase-loan

bethpotts committed:
SME feedback: purchase-loan

bethpotts committed:
SME feedback: irrrl

Commit:
Merge remote-tracking branch 'origin/content/wip/housing-assistance' into housing-template-updates

melwoodard committed:
Merge pull request #5622 from department-of-veterans-affairs/housing-template-updates

Housing template updates

DanielleSoCompany committed:
Updates based on SME changes

Commit:
Updating filename per slack

DanielleSoCompany committed:
updating URLS per filename change

Commit:
Updating folder name to match content change

DanielleSoCompany committed:
updating per SME changes to NADL option

DanielleSoCompany committed:
updates based on SME feedback to NADL option

DanielleSoCompany committed:
updates based on SME feedback to NADL option

DanielleSoCompany committed:
updates based on SME feedback to NADL option

DanielleSoCompany committed:
updating loan limit $ per va.gov

Beth: As I was looking into the loan limit info per SME feedback, I saw that the $ had been updated for 2017. So I've updated it here. It's now $424,100 (up from $417,000).

DanielleSoCompany committed:
updating loan limit $ per Va.gov

Beth: As I was looking into the loan limit information on VA.gov, I noticed that it's been updated for 2017. It's now $424,100 (up from $417,000). I've updated here to match.

DanielleSoCompany committed:
adding loan limit bullet per SME feedback

DanielleSoCompany committed:
adding loan limit bullet per SME feedback

DanielleSoCompany committed:
adding sentence re: loan limits per SME feedback

DanielleSoCompany committed:
updating per SME feedback

Beth: Here, I needed to walk a line with the SME feedback on "because of the VA partial guarantee..." because of the fact that we've added "VA direct and VA-backed home loans." So I've revised to say "But you may be offered better loan terms than with a traditional loan from a private bank or mortgage company." And then will address full SME comment on the "Loan Options" page within VA-backed loans.

DanielleSoCompany committed:
updates per SME feedback

DanielleSoCompany committed:
Create loan-options.md

DanielleSoCompany committed:
adding related links header

Beth: Tried to differentiate this header a little from the main page title.

DanielleSoCompany committed:
updates per SME feedback on NADL option

DanielleSoCompany committed:
1 last update per SME feedback

bethpotts committed:
SME feedback: cash-out-refinance

bethpotts committed:
SME feedback: adaptive-housing-grants

bethpotts committed:
SME feedback: housing-assistance landing page

bethpotts committed:
SME feedback: adaptive-housing-grants

bethpotts committed:
SME feedback: nadl

Commit:
Add accordions to COE page

bethpotts committed:
Added text above new accordions: apply-for-coe

bethpotts committed:
Adjusting accordion text: apply-for-coe

bethpotts committed:
Formatting tweaks to accordions: apply-for-coe

bethpotts committed:
Final proof: adaptive-housing-grants

bethpotts committed:
Final proof: apply-for-certificate-of-eligibility

Changed eBenefits link to bring Vets to a more specific page rather than the general landing page.

bethpotts committed:
Final proof: eligibility

Commit:
Merge branch 'master' into content/wip/housing-assistance

bethpotts committed:
Final proof: loan-options

bethpotts committed:
Final proof: nadl

bethpotts committed:
Final proof: purchase-loan

melwoodard committed:
Update loan-options collection to point to new URL

bethpotts committed:
Proof: loan-options

bethpotts committed:
Final proof: home-loans

melwoodard committed:
Merge branch 'master' into content/wip/housing-assistance

DanielleSoCompany committed:
incorporating SME round 2 change

DanielleSoCompany committed:
Incorporating SME round 2 change

DanielleSoCompany committed:
Incorporating SME Round 2 comment

DanielleSoCompany committed:
Incorporating SME Round 2 changes

DanielleSoCompany committed:
Incorporating SME Round 2 changes

DanielleSoCompany committed:
Incorporating SME Round 2 changes

DanielleSoCompany committed:
Incorporating SME Round 2 changes

DanielleSoCompany committed:
Incorporating SME Round 2 changes

DanielleSoCompany committed:
Incorporating SME Round 2 changes

DanielleSoCompany committed:
Incorporating additional SME Round 2 feedback

DanielleSoCompany committed:
Incorporating additional SME Round 2 changes

DanielleSoCompany committed:
Incorporating additional SME Round 2 changes

DanielleSoCompany committed:
Incorporating SME Round 2 changes

DanielleSoCompany committed:
Fixing changes after heroku preview

DanielleSoCompany committed:
Updates post heroku preview

DanielleSoCompany committed:
Incorporating SME Round 2 changes

DanielleSoCompany committed:
Adding <br>s to fix spacing

DanielleSoCompany committed:
Post-heroku preview changes

DanielleSoCompany committed:
Additional update to SME changes

Beth: Made the lead-ins under "Why might I want a VA-backed purchase loan?" h4 headers because the text felt like was getting a bit lost. I believe Mel had done this on another similar page in another section of the site and I thought it looked much better. If you disagree, please feel free to change back!

DanielleSoCompany committed:
Additional post-heroku preview changes

DanielleSoCompany committed:
spacing fixes

DanielleSoCompany committed:
Additional post-heroku preview updates

DanielleSoCompany committed:
Additional post-heroku preview changes

DanielleSoCompany committed:
Post-heroku preview update

DanielleSoCompany committed:
Breaking up PMI and MIP bullets for bullet length

DanielleSoCompany committed:
Removing period in bullet

DanielleSoCompany committed:
Removing period in bullet

DanielleSoCompany committed:
Adding link to find regional loan center

bethpotts committed:
Proof tweak: loan-options

bethpotts committed:
Proof tweak: purchase-loan

bethpotts committed:
Proof tweak: irrrl

DanielleSoCompany committed:
Adding <br> for spacing

DanielleSoCompany committed:
proofing update

bethpotts committed:
Add housing home page blurb and link

bethpotts committed:
switch order of home page blurbs

DanielleSoCompany committed:
proofing updates

DanielleSoCompany committed:
proofing change

DanielleSoCompany committed:
proofing updates

DanielleSoCompany committed:
proofing update

DanielleSoCompany committed:
proofing update

DanielleSoCompany committed:
Proofing updates

Beth: I removed "your" from "Get your military service records online" link because it's under the spouse accordion. So they wouldn't be getting their records; they'd be getting their spouse's records. Just bringing to your attention for an accuracy check. Also, I put the "Download Form 21-534" on its own close to the second bullet since it's directly tied to that one, and then put "Find out if you qualify..." and "Get military service records" links slightly below with another break since those 2 apply to both conditionals in the bullets.

robertfairhead committed:
Merge branch 'master' into content/wip/housing-assistance

rroueche commented:
@melwoodard @amybehrens @bethpotts is this good to go?

bethpotts reviewed (approved):
SME feedback incorporated. Proof complete. Ship!

Commit:
Adding housing assistance to nav

Commit:
Lower case l in Home loan L2 nav

rroueche committed:
Merge branch 'master' into content/wip/housing-assistance

Commit:
Merge branch 'master' into content/wip/housing-assistance

rroueche commented:
@amybehrens @bethpotts @DanielleSoCompany merging now!
